benches/benches/bevy_ecs/entity_cloning.rs

```diff
@@ -77,7 +77,7 @@ fn reflection_cloner<B: Bundle + GetTypeRegistration>(
     for component in component_ids {
         builder.override_clone_behavior_with_id(component, ComponentCloneBehavior::reflect());
     }
-    builder.recursive(recursive);
+    builder.linked_cloning(recursive);
```

Comment:
should the `recursive` variable be renamed?

Reply:
Yup!

Reply:
Yup!